src/org/kms/core/KMSUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.kms.core;
+
+import java.util.HashSet;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+public class KMSUtils {
+
```

Comment:
I have used Utils because in future if we need any class variables and class methods, we can put it in Utils.java